NEWS.md

```diff
@@ -1,8 +1,12 @@
 # htmltools (development version)
 
+## Improvements
```

Comment:
I went back and forth on putting this in "possible breaking changes". It's more likely this change will un-break unintentionally broken styles and less likely to break working styles, so I settled on calling it an "improvement".